app/src/main/AndroidManifest.xml

```diff
@@ -1,23 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.dispatcher">
+    package="com.example.dispatcher" >
 
     <application
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_dispatcher_logo"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@drawable/ic_dispatcher_logo"
         android:supportsRtl="true"
         android:theme="@style/Theme.Dispatcher">
         <activity
-            android:name=".MainActivity"
+            android:name=".SplashScreenActivity"
```

Comment:
Does the Splash an activity for the user?
Lets refactor SplashScreenActivity to be a fragment that is shown first and then replace by a main fragment